src/synopsys-action/tools-parameter.ts

```diff
@@ -288,7 +288,7 @@ export class SynopsysToolsParameter {
     const githubRepoName = githubRepo !== undefined ? githubRepo.substring(githubRepo.indexOf('/') + 1, githubRepo.length).trim() : ''
     const githubBranchName = parseToBoolean(inputs.POLARIS_PRCOMMENT_ENABLED) ? process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_HEAD_REF] : process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REF_NAME]
     const githubRef = process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_REF]
-    const githubAPIURL = process.env[GITHUB_ENVIRONMENT_VARIABLES.GITHUB_API_URL]
+    const githubHostUrl = inputs.GITHUB_HOST_URL ? inputs.GITHUB_HOST_URL : ''
```

Comment:
We should default this value otherwise this will fail when
User is not passing the host url when they are using github enterprise edition

Reply:
this is expected behaviour, github host url needs to be configured by users